test/test_bdv_datasets.py

```diff
@@ -0,0 +1,183 @@
+import os
+import unittest
+from abc import ABC
+from shutil import rmtree
+
+import numpy as np
+from pybdv import make_bdv, BdvDataset
+from pybdv.util import open_file, get_key, HDF5_EXTENSIONS
+
+try:
+    import z5py
+except ImportError:
+    z5py = None
+
+DOWNSCALE_MODE = 'interpolate'
+
+
+class UtilTestMixin(ABC):
```

Comment:
Could you change the name of the test classes so that it's clear they are for the `BdvDataset`?

Reply:
Done